.gitignore

```diff
@@ -31,3 +31,4 @@ pnpm-debug.log.*
 yarn-error.log
 yarn-error.log.*
 .env
+.idea/
```

Comment:
Do you think it would be good enough to add this on your side in [`~/.config/git/ignore`](https://git-scm.com/docs/gitignore)?

Reply:
yeah, let's scratch this idea.